packages/react-components/src/organisms/DiscussionModal.tsx

```diff
@@ -97,22 +112,22 @@ const QuickCheckReplyModal: React.FC<QuickCheckReplyModalProps> = ({
               {crossIcon}
             </Button>
           </div>
-          <Headline3>Reply to quick check</Headline3>
+          <Headline3>{title}</Headline3>
         </header>
         <div css={[paddingStyles, { paddingTop: 0 }]}>
           <Controller
-            name="replyText"
+            name={discussionType}
```

Comment:
if we change this `DiscussionModalData` to be `{ text: string }`

```suggestion
            name="text"
```